build-tests/api-documenter-test/config/api-extractor.json

```diff
@@ -1,12 +1,15 @@
 {
-  "$schema": "https://developer.microsoft.com/json-schemas/api-extractor/v7/api-extractor.schema.json",
+  "$schema": "../../../apps/api-extractor/src/schemas/api-extractor.schema.json",
```

Comment:
Question for reviewers: without this change, we cannot add tests that utilize the updated schema. Is there a preferred alternate pattern for doing this? Or is this right?

Reply:
It's fine. API Extractor (and Rush Stack tools in general) ignore the `$schema` field, treating it as a directive for editors like VS Code.  The tools bundle their own JSON schema, because that is the only way to ensure deterministic behavior.

Reply:
Ah, okay. I can revert these then. Note that the `ExtractorConfig` unit tests do appear to respect this, though. My unit tests were failing schema validation until I changed them to use the local file path.